src/components/app/game-board/side-panel/game-status/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styles from './ProgressBar.module.scss';
+
+export default function ProgressBar({percentage, color}: { percentage: number, color: string }): React.JSX.Element {
+    return (
+        <div className={`${styles['progress-bar-container']} w-full bg-gray-200 rounded-full h-1.5 mb-4 dark:bg-gray-700`}>
```

Comment:
If all been taken from a specific that is only for the  progress-bar do we need the prefix ? Consider changing the name of the import so it will progressBarStyle['container']